src/register/constants/organisationHaveSra.ts

```diff
@@ -0,0 +1,57 @@
+export const organisationHaveSra = {
+  formValues:{},
+  meta: {
+    idPrefix: 'tbc',
+    name: 'sra',
+    header: "Do you have an SRA number?",
+    formGroupValidators: [],
+    validationHeaderErrorMessages: [
```

Comment:
You seem to have copied the older version of this file. Check the other (deleted) `organisationHaveSra.ts` file here.